src/hotspot/share/cds/cds_globals.hpp

```diff
@@ -71,6 +71,11 @@
   product(bool, AllowArchivingWithJavaAgent, false, DIAGNOSTIC,             \
           "Allow Java agent to be run with CDS dumping")                    \
                                                                             \
+  develop(ccstr, ArchiveHeapTestClass, NULL,                                \
+          "For JVM internal testing only. Not available in "                \
+          "product builds. The \"archivedObjects\" static field "           \
+          "of the specified class is stored in the CDS archive heap")       \
+                                                                            \
```

Comment:
"Not available in product builds" is redundant with the flag definition.

Reply:
How about "The static field named \"archivedObjects\" of the specified class is stored in the CDS archive heap.

Reply:
Fixed